README.md

```diff
@@ -1,86 +1,61 @@
 # KubeVirt Project Infrastructure Tools
 
-This repository provides supporting code for the project infrastructure.
+This repository provides supporting code for the project infrastructure. This is
+what you can find on each of the repo directories
 
- * cni-plugins
+* `cni-plugins`: code to deploy CNI plugins (currently only sriov-passthrough-cni
+is available)
 
- Code to deploy CNI plugins (currently only sriov-passthrough-cni is available)
+* `docs`: extended documentation of several aspects of our infrastructure
 
- * external-plugins
+* `external-plugins`: Prow plugins used on our setup
 
- Prow plugins used on our setup
+* `github/ci`: Infrastructure code for our main deployments:
 
- * github/ci
+  * `github/ci/phx-prow`: Ansible code for provisioning phx-prow cluster
 
- Infrastructure code for our main deployments:
+  * `github/ci/prow-deploy`: Ansible code for testing and deploying Prow components,
+  includes Prow configuration under github/ci/prow-deploy/files
 
-   * github/ci/phx-prow
+  * `github/ci/prow-workloads`: Ansible code for bootstrapping prow-workloads cluster
+  with Kubespray
 
-   Ansible code for provisioning phx-prow cluster
+  * `github/ci/services`: Code to manage additional CI services
 
-   * github/ci/prow-deploy
+  * `github/ci/testgrid`: Code to manage the configuration for our
+  [testgrid setup](https://testgrid.k8s.io/kubevirt)
 
-   Ansible code for testing and deploying Prow components, includes Prow configuration under github/ci/prow-deploy/files
+* `images`: Definition of container images used in CI
 
-   * github/ci/prow-workloads
+* `limiter`: Tool used to control connections for GCE buckets to the outside world
+depending on billing alerts. See [README](limiter/README.md)
 
-   Ansible code for bootstraping prow-workloads cluster with Kubespray
+* `releng/release-tool`: Tool for creating KubeVirt releases
 
-   * github/ci/services
+* `robots`: Several Go based automation tools
```

Comment:
Is the language in which they are written is important to be mentioned here?

Reply:
Not at all, removed.